src/common/snippets/include/snippets/utils/utils.hpp

```diff
@@ -85,6 +86,8 @@ inline auto normalize_rank(int32_t allocation_rank, const size_t shape_rank) ->
     return allocation_rank < 0 ? allocation_rank + static_cast<int32_t>(shape_rank) + 1 : allocation_rank;
 }
 
+std::optional<int64_t> get_softmax_axis_last_dim(const std::shared_ptr<const ov::Node>& node);
```

Comment:
1. `last_dim` postfix seems unnecessary: we don't have any last_dim specific logic inside the helper
2. Let's add some brief docs to this method: we need at least to explain in what cases nullopt is returned